packages/fx-core/templates/core/v3Migration/js.ts.app.yml

```diff
@@ -130,6 +139,7 @@ deploy:
       execPath: $\{{ {{~dotnetPath~}} }} # Use dotnet installed by prerequisite/install action
```

Comment:
Did you miss this one regarding adding a name for `cli/runXXXCommand`?

Reply:
I added a name to a driver that appeared in the same lifecycle more than once.